tests/Posts/Year2017/ListeningNetteComponents/Presenter/CategoryPresenter.php

```
@@ -57,6 +58,8 @@ public function __construct(
         BasketContentControlFactoryInterface $basketContentControlFactory,
         EventDispatcherInterface $eventDispatcher
     ) {
+        parent::__construct();
```

ok, I'll remove the call and add this to ignored errors